modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProvider.java

```diff
@@ -152,6 +153,42 @@ public HistoricalMetric getHistoricalViewsHistoricalMetric(
 		}
 	}
 
+	public List<ReferringSocialMedia> getReferringSocialMediaList(
+			long companyId, String url)
+		throws PortalException {
+
+		try {
+			String response = _asahFaroBackendClient.doGet(
+				companyId,
+				"api/1.0/pages/social-page-referrers?canonicalURL=" + url +
+					"&interval=D&rangeKey=30");
+
+			TypeFactory typeFactory = _objectMapper.getTypeFactory();
+
+			Map<String, Long> socialTrafficChannels = _objectMapper.readValue(
```

Comment:
This name sounds a bit estrange for me, What do you think about `socialPageReferrers`?